tests/development/messages/content_types/test_406_errors.py

```diff
@@ -0,0 +1,43 @@
+import requests
```

Comment:
Same, content negotiation is across the entire API, its not specific to an endpoint